glossary-sync.py

```diff
@@ -188,26 +177,32 @@ def _glossary_term_from_datahub(
 def fetch_datahub_glossary():
     graph = get_graph()
 
-    # Get all the urns in the glossary.
+    logger.info("Get all the urns in the glossary.")
     urns = list(graph.get_urns_by_filter(entity_types=["glossaryTerm", "glossaryNode"]))
+    logger.info(f"Got {len(urns)} urns")
 
-    # Hydrate them into entities.
-    entities = {urn: graph.get_entity_semityped(urn) for urn in urns}
+    logger.info("Hydrate them into entities.")
+    entities = {}
+    for urn in progressbar.progressbar(urns):
+        try:
+            entities[urn] = graph.get_entity_semityped(urn)
+        except requests.exceptions.HTTPError as e:
+            logger.error(f"Error for urn {urn}: {e}")
```

Comment:
Why can this fail? Should we retry in this case or it is fine if we just log a failure?

Reply:
Fine to log error. This is corner cases for URN encoding which will need to resolved